static/js/components/inputs/FieldsOfStudySelectField.js

```diff
@@ -1,42 +1,36 @@
 // @flow
 import React from 'react';
 import _ from 'lodash';
+
 import SelectField from './SelectField';
-import {
-  caseInsensitiveFilter,
-  showLimitedOptions,
-  highlightMatchedOptionText
-} from '../utils/AutoCompleteSettings';
 import FIELDS_OF_STUDY from '../../data/fields_of_study';
+import type { Validator, UIValidator } from '../../lib/validation/profile';
+import type {
+  Profile,
+  ValidationErrors,
+  UpdateProfileFunc,
+} from '../../flow/profileTypes';
 
 let fieldOfStudyOptions = _.map(FIELDS_OF_STUDY, (name, code) => ({
   value: code,
   label: name
 }));
 
-export default class FieldsOfStudySelectField extends React.Component {
-  static propTypes = {
-    maxSearchResults: React.PropTypes.number
-  };
-
-  static defaultProps = {
-    maxSearchResults: 10
-  };
-
-  static autocompleteStyleProps = {
-    menuStyle: {maxHeight: 300},
-    menuHeight: 300,
-    listStyle: {width: '100%'},
-    fullWidth: true
-  };
+type FieldOfStudySelectProps = {
+  className:                  string,
+  errors:                     ValidationErrors,
+  keySet:                     Array<string>,
+  label:                      string,
+  onChange:                   Function,
+  profile:                    Profile,
+  updateProfile:              UpdateProfileFunc,
+  updateValidationVisibility: (xs: Array<string>) => void,
+  validator:                  Validator|UIValidator,
+};
 
-  render() {
-    return <SelectField
-      options={fieldOfStudyOptions}
-      autocompleteStyleProps={FieldsOfStudySelectField.autocompleteStyleProps}
-      autocompleteBehaviors={[showLimitedOptions, highlightMatchedOptionText]}
-      filter={caseInsensitiveFilter}
-      {...this.props}
-    />;
-  }
-}
+export default (props: FieldOfStudySelectProps) => (
```

Comment:
yeah definitely, I kept this around just due to precedent but I'm more than happy to remove it.